opal/mca/hwloc/hwloc-internal.h

```diff
@@ -203,8 +203,8 @@ typedef uint16_t opal_binding_policy_t;
  * cpu list based on its node-local-rank */
 #define OPAL_BIND_ORDERED 0x8000
 
-/* binding policies - any changes in these
- * values must be reflected in orte/mca/rmaps/rmaps.h
+/*
```

Comment:
these may effectively be dead code at this point; we should probably start trying to strip down the hwloc-internal.h header.  Some left-over 3rd-party work :(.